platform/viewer/src/routes/Local/Local.tsx

```diff
@@ -60,11 +60,38 @@ function Local() {
   const firstLocalDataSource = localDataSources[0];
   const dataSource = firstLocalDataSource.createDataSource({});
 
+  const microscopyExtensionLoaded = extensionManager.registeredExtensionIds.includes(
+    '@ohif/extension-dicom-microscopy'
+  );
+
   const onDrop = async acceptedFiles => {
     const studies = await filesToStudies(acceptedFiles, dataSource);
-    // Todo: navigate to work list and let user select a mode
+
     const query = new URLSearchParams();
+
+    if (microscopyExtensionLoaded) {
+      const smStudies = studies.filter(id => {
+        const study = DicomMetadataStore.getStudy(id);
+        return (
+          study.series.findIndex(
+            s => s.Modality === 'SM' || s.instances[0].Modality === 'SM'
+          ) >= 0
+        );
+      });
+
+      if (smStudies.length > 0) {
+        smStudies.forEach(id => query.append('StudyInstanceUIDs', id));
+
+        navigate(
```

Comment:
please add a Todo that:
 we are forcing to go to `microscopy` mode which is not ideal, we should make the local drag and drop navigate to the worklist and there user can select microscopy mode

Reply:
How about something similar to this

```js
let studies = await filesToStudies(acceptedFiles, dataSource)
let queryParams = ""

if (microscopyExtensionLoaded) {
  const smStudies = studies.filter((id) => {
    const study = DicomMetadataStore.getStudy(id)
    return (
      study.series.findIndex(
        (s) => s.Modality === "SM" || s.instances[0].Modality === "SM"
      ) >= 0
    )
  })

  if (smStudies.length > 0) {
    studies = smStudies
    queryParams = "microscopy"
  }
}

const studyInstanceUIDs = studies
  .map((id) => `StudyInstanceUIDs=${id}`)
  .join("&")
const url = `/${queryParams}/dicomlocal?${studyInstanceUIDs}`

navigate(url)
```